myproject/tests.py

```diff
@@ -1,29 +1,58 @@
-import unittest
+import pytest
 
 from pyramid import testing
 
+ITEMS_TO_CHECK = [
+    'title',
+    'creation_date',
+    'body'
+]
 
-class ViewTests(unittest.TestCase):
-    def setUp(self):
-        self.config = testing.setUp()
 
-    def tearDown(self):
-        testing.tearDown()
+def test_list_view():
+    from .views import list_view
+    request = testing.DummyRequest()
+    info = list_view(request)
+    assert 'title' in info
 
-    def test_my_view(self):
-        from .views import my_view
-        request = testing.DummyRequest()
-        info = my_view(request)
-        self.assertEqual(info['project'], 'MyProject')
 
+@pytest.mark.parametrize('value_to_return', ITEMS_TO_CHECK)
+def test_detail_view(value_to_return):
+    from .views import detail_view
+    request = testing.DummyRequest()
+    request.matchdict = {'id': '11'}
+    info = detail_view(request)
+    assert value_to_return in info
 
-class FunctionalTests(unittest.TestCase):
-    def setUp(self):
-        from myproject import main
-        app = main({})
-        from webtest import TestApp
-        self.testapp = TestApp(app)
 
-    def test_root(self):
-        res = self.testapp.get('/', status=200)
-        self.assertTrue(b'Pyramid' in res.body)
+@pytest.mark.parametrize('value_to_return', ITEMS_TO_CHECK)
+def test_editl_view(value_to_return):
```

Comment:
this test doesn't appear to have much to do with editing.  why does it have this name?